server/routes/makeAReferral/risk-information/oasys/view/oasysRiskInformationView.ts

```diff
@@ -42,7 +42,7 @@ export default class OasysRiskInformationView {
   private get confirmUnderstoodWarningCheckboxArgs(): CheckboxesArgs {
     return {
       idPrefix: 'confirm-understood',
-      name: 'confirm-understood[]',
```

Comment:
the two changes in this file seem strange - where did the old values come from?

Reply:
I think this was a problem caused from when I was rebasing.
I will remove it from the first commit.